src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
```

Comment:
I agree. What about making it similar to angular-seed project

.../components/errorhandling
.../components/i18n

more structure and it is not too easy to just dump code into a central place